src/main/kotlin/io/thecontext/ci/value/Person.kt

```diff
@@ -10,6 +10,9 @@ data class Person(
         @JsonProperty("name")
         val name: String,
 
+        @JsonProperty("email")
+        val email: String?,
```

Comment:
Isn't `Person` also used for guests and not only for hosts? I'm wondering if introducing email on `Person` just for having an itunes owner email in the feed makes sense. I see this more as a config property that belongs to `podcast.yaml`. On the other hand, we might want to have an email address in the show notes (similar to link to twitter account and website) as well, but this use case is not covered by this PR.

So I'm not 100% sure if I like that idea or not. Your call @ming13 . I'm fine if you would like to keep `Person.email` I'm just saying that it feels a bit overhead just to cover the itunes owner usecase and it might be a bit missleading what `Person.email` is acutally used for (especially if it is not used in the show notes for linking like we do with twitter or website).

Reply:
Yes, it does feel a bit forced, have to agree on that. However, the current `podcast.yml` works with people IDs... Let’s see how it goes and we can refine from that.